common/components/chrome/SectionLink.jsx

```diff
@@ -6,6 +6,7 @@ import cx from '../utils/cx';
 import NavigationDispatcher from '../stores/NavigationDispatcher.js';
 import React from 'react';
 import url from '../utils/url.js';
+import Section from '../enums/Section.js';
```

Comment:
Is this being used?  It looks like we're actually missing the import for SectionType.

Reply:
I haven't messed with this file/these files.  The exports in enums/Section.js look a little strange. My linter goes nuts over the flow syntax so it's hard to tell, but it does indicate that Section isn't used in SectionLink.jsx.